README.md

```diff
@@ -42,10 +42,15 @@ be specified:
 * `FOREMAN_USE_ORGANIZATIONS`: whether to use organizations or not (value can be true/false)
 * `FOREMAN_USE_LOCATIONS`: whether to use locations or not (value can be true/false).
 
-*PuppetLabs Puppet installation test*
+*Foreman Puppet integration test: fb-puppet-tests.bats*
```

Comment:
It's not very consistent to give the exact bats name. I don't mind it, but I'd either do it everywhere or nowhere.